backend/windmill-api-client/bundled.json

```diff
@@ -0,0 +1,7480 @@
+{
```

Comment:
I think we should .gitignore this file too since it's auto-generated and prone to get out of sync.